src/kGenerator.js

```diff
@@ -14,9 +15,47 @@ module.exports = {
 	},
 	generate: function (opts) {
 		//opts will be a set of key value functions
+		var wstream = fs.createWriteStream('demo/itOutput.js');
+		wstream.write("/*------------Test cases scaffolding----------------*/\n\n\r");
+		const itGen = new it(wstream);
+
 		var ast = csstree.parse(this.cssFile);
-		if (ast.type != "StyleSheet") {
-			throw new Error("File is not StyleSheet format");
+
+		if (ast.stylesheet) {
+
+			ast.stylesheet.rules.forEach(function (rule) {
+
+				if (rule.type == "rule") {
+					rule.selectors.forEach(function (selector) {
+						console.log("selector: ", selector);
```

Comment:
shouldn't we use the `debug` module?
console.log is synchronous and cannot be disabled from external